Decred Wallet/Features/Settings/SettingsController.swift

```diff
@@ -62,12 +83,34 @@ class SettingsController: UITableViewController  {
         }
     }
     
+    func setMenuTitle(){
+        self.changeSpendingPinPassDesc.text = LocalizedStrings.changeSpendingPinPassDesc
```

Comment:
With reference to https://github.com/raedahgroup/dcrios/pull/470/files#r296964559, what runtime error do you get when you specify localized string keys for these labels? Please share with pastebin.

Reply:
fixed